integration/tests/select-template.test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from '@esm-bundle/chai';
```

Comment:
Should also remove `packages/select/test/template.test.js`

Reply:
Thanks, removed.